README.md

```diff
@@ -2,3 +2,13 @@
 This repository is used to document and define coding practices such as coding styles, writing commit messages and documentation.
 
 The code_conventions.md defines the general coding conventions, without going into language specifics. It is possible that certain languages have their own coding standards. If this is the case, an additional document is added, which provides addendums or changes to the general codestyle.
+
+Vocabulary
```

Comment:
I think this vocabulary section should be put in a separate file. It could become quite significant.

Reply:
Sure, but I did make the explicit choice to add it to the readme to prevent ending up with a load of different files describing different meta-agreements.